__init__.py

```diff
@@ -1274,6 +1354,12 @@ def do_stop(self):
 
     def stop(self):
         """ Stop playback. """
+        # If authorized and user has set config allowing skill to control
+        # playback elsewhere, the schedule_event approach to stop spotify may
+        # not work, but pause should work regardless of which device is playing
+        if self.spotify and self.allow_master_control:
+            self.pause()
+            return True
```

Comment:
Should this check if a unit is playing before performing the pause and returning true?

In this case the skill will always consume the stop event if it's configured and for example the button press for triggering listening on the Mark-1 will never be triggered.

Reply:
Oh yes, good point, I think you're right that it should have a is_playing check in there.